tests/php/validation/test-class-amp-validation-manager.php

```diff
@@ -522,8 +522,8 @@ public function test_add_validation_error_sourcing_gutenberg() {
 		if ( ! function_exists( 'do_blocks' ) ) {
 			$this->markTestSkipped( 'Gutenberg not active.' );
 		}
-		if ( version_compare( get_bloginfo( 'version' ), '4.9', '<' ) ) {
-			$this->markTestSkipped( 'The WP version is less than 4.9, so Gutenberg did not init.' );
+		if ( version_compare( get_bloginfo( 'version' ), '5.0', '<' ) ) {
+			$this->markTestSkipped( 'Gutenberg for WP < 5.0 is not supported.' );
```

Comment:
Is this change and the latter correct since in WP 4.9 the 4.9.0 version of Gutenberg is installed?

Reply:
I meant 'not supported' in the sense that the `do_blocks` hook occurs at priority 7 instead of 9 in GB 4.9.0, so the functionality being tested here would fail.